src/ac_training_lab/pioreactor/on_reactor.py

```diff
@@ -0,0 +1,321 @@
+import requests
+import json
+from datetime import datetime
+import paho.mqtt.client as mqtt
+import json
+import lookhere
+import time
+
+# Author: Enrui (Edison) Lin 
+
+
+# The lookhere.py file contains the following variables:
+# broker = "your_broker_address"
+# port = 8883
+# username = "your_username"
+# password = "your_password"
+# Both lookhere.py and this script should be in the same directory on the Pioreactor.
+
+# This script should be run on the Pioreactor. It listens for commands from the MQTT broker and interacts with the Pioreactor API accordingly.
+
+# It should be run with: python on_reactor.py >/dev/null 2>&1 & disown
```

Comment:
Thanks for including these instructions - very useful. It would be nice to ensure the MQTT script runs upon startup on the Raspberry Pi. My guess is "probably", though it is a custom OS. Thoughts? If it's not obvious/straightforward, maybe ask on the forum.

Reply:
We may be able to add it to a bashrc launch file, not sure though. Will try later.

Reply:
Moving this discussion to https://github.com/AccelerationConsortium/ac-training-lab/issues/72

Reply:
crontab maybe preferred: https://github.com/AccelerationConsortium/ac-training-lab/issues/72#issuecomment-2394725701